sdk/src/beta9/cli/config.py

```diff
@@ -175,3 +179,38 @@ def select_context(name: str, config_path: Path):
     save_config(contexts=contexts, path=config_path)
 
     terminal.success(f"Default context updated with '{name}'.")
+
+
+@management.command(
+    name="export-config",
```

Comment:
Can we name this just "export"? I think this command being under the config subcommand is enough to say what kind of export this is. e.g. `beam config export`